extensions/sql-migration/src/api/sqlUtils.ts

```diff
@@ -49,7 +49,7 @@ const query_databases_with_size = `
 		db_size
 		AS
 		(
-			SELECT database_id, CAST(SUM(size) * 8.0 / 1024 AS INTEGER) size
+			SELECT database_id, CAST(SUM(size) / 128 AS bigint) size
```

Comment:
it is still MB?

Reply:
Yes, 1024 = 128 * 8. The 8s cancel each other out and we have one operation less.

Reply:
better to leave a comment to improve readability.